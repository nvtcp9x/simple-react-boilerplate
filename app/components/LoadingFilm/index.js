import React from 'react';

export default () => (<div>Waiting for films...</div>)
