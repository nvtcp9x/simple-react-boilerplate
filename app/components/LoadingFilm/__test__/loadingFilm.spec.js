import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import LoadingFilm from '../index';

Enzyme.configure({ adapter: new Adapter() });

describe('LoadingFilm component', () => {
  const component = shallow(<LoadingFilm />);

  it('should ...', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
