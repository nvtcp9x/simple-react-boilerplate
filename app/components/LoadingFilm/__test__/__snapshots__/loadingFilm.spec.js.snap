// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingFilm component should ... 1`] = `
<div>
  Waiting for films...
</div>
`;
